app/build.gradle

```diff
@@ -47,16 +46,16 @@ android {
 dependencies {
     // There is a better way to declare and structure dependencies.
     // However, that is a different topic.
-    def daggerVersion = '2.14'
-    def butterKnifeVersion = '8.8.0'
-    def kotlinVersion = '1.2.0'
+    def daggerVersion = '2.14.1'
+    def butterKnifeVersion = '8.8.1'
+    def kotlinVersion = '1.2.21'
```

Comment:
The newest version is 1.2.30. However, Android Studio 3.2.0 includes Kotlin plugin v1.2.21.